src/DDTrace/Integrations/Lumen/V5/LumenIntegrationLoader.php

```diff
@@ -45,7 +46,8 @@ public function load()
                 $resourceName = $this->currentRoute[1]['as'];
             }
             if (null !== $resourceName) {
-                $span->setTag(Tag::RESOURCE_NAME, $span->getTag(Tag::HTTP_METHOD) . ' ' . $resourceName);
+                $operationName = $span->getTag(Tag::HTTP_METHOD) . ' ' . $resourceName;
```

Comment:
same http method request.

Reply:
I only updated the already existing `resource name` tag as was from upstream. Are you requesting I change that?

Reply:
My request is that http methods aren't used as prefixes in operation names.  It would be good to confirm that cardinality is under control as well.